components/omnibox/browser/brave_omnibox_edit_model.cc

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) 2023 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#include "brave/components/omnibox/browser/brave_omnibox_edit_model.h"
+
+#include "brave/components/commander/common/commander_frontend_delegate.h"
+#include "brave/components/commander/common/commander_url.h"
+#include "components/omnibox/browser/omnibox_edit_model.h"
+
+BraveOmniboxEditModel::~BraveOmniboxEditModel() = default;
+
+void BraveOmniboxEditModel::OpenMatch(
```

Comment:
it just occurred to me, maybe we should not trigger this for "paste and go" type inputs because the user might be pasting malicious input?

Reply:
Good call!